avocado/utils/pci.py

```diff
@@ -182,7 +182,9 @@ def get_slot_from_sysfs(full_pci_address):
     if not os.path.isfile("/proc/device-tree/%s/ibm,loc-code" % devspec):
         return
     slot = genio.read_file("/proc/device-tree/%s/ibm,loc-code" % devspec)
-    slot = re.match(r'((\w+)[.])+(\w+)-P(\d+)-C(\d+)|Slot(\d+)', slot).group()
+    slot_re = re.match(r'((\w+)[.])+(\w+)-P(\d+)-C(\d+)', slot)
+    if slot_re:
+        return slot_re.group()
```

Comment:
I cannot test this. Could you please provide also a selftest with an example content for `ibm,loc-code`?

Reply:
@apahim Can you help me with what exact self test you are expecting in this case please ?